rickchurch/constants.py

```diff
@@ -3,18 +3,21 @@
 from decouple import config
 
 
+DISCORD_BASE_URL: str = "https://discord.com/api"
+
 log_level: str = config("LOG_LEVEL", default="INFO")
 
 base_url: str = config("BASE_URL")  # URL to host church of rick
-discord_token_url: str = config("TOKEN_URL", default="https://discord.com/api/oauth2/token")
-discord_user_url: str = config("USER_URL", default="https://discord.com/api/users/@me")
 
 # Get these from https://discord.com/developers/applications, OAuth2 section
 client_id: str = config("CLIENT_ID")
 client_secret: str = config("CLIENT_SECRET")
 # Get this by adding {base_url}/oauth_callback as URI in the application's OAuth2 section
 oauth_redirect_url: str = config("OAUTH_REDIRECT_URL")
 
+discord_guild_id: str = config("DISCORD_GUILD_ID", default="849299937485127701")
```

Comment:
There is no reason to hard-code our guild ID, we should only hard-code values that are likely not to be changed by anyone using this. This isn't one of those variables, since if someone wanted to run this for testing purposes, he wouldn't get an error of not setting this variable, yet it would cause further errors later, since his application bot wouldn't be joined in our server. It is better to keep this without a default value.

```suggestion
discord_guild_id: str = config("DISCORD_GUILD_ID")
```